lib/vdsm/network/link/setup.py

```diff
@@ -20,38 +20,64 @@
 from __future__ import absolute_import
 from __future__ import division
 
-from vdsm.network.link.bridge import Bridge
+
+class NmstateBridgeOptionNotSupported(Exception):
+    pass
 
 
 def parse_bond_options(options):
     """
     Parse bonding options into a dictionary.
     """
-
-    def _string_to_dict(str, div, eq):
-        if options == '':
-            return {}
-        return dict(
-            option.split(eq, 1) for option in str.strip(div).split(div)
-        )
-
     if options:
         d_options = _string_to_dict(options, ' ', '=')
         return d_options
     else:
         return {}
 
 
-def setup_custom_bridge_opts(nets):
-    for name, opts in parse_nets_bridge_opts(nets):
-        Bridge(name, opts)
+def parse_nets_bridge_opts(opts):
 
+    nmstate_options_types = {
```

Comment:
This is still a constant, it should have capital name and better place is under imports.

Reply:
the problem was when it was is under imports it didn't recognize `str_to_bool` func. anyway, i think now it's more readable and well encapsulated.